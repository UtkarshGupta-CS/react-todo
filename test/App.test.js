import React from 'react';
import renderer from "react-test-renderer";
import App from "../src/components/App";
describe("Todo component renders the todo correctly", () => {
  it("renders correctly", () => {
    const rendered = renderer.create(<App />);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
