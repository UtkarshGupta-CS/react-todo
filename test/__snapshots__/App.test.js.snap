// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component renders the todo correctly renders correctly 1`] = `
<div
  className="sc-EHOje ezyMVi"
>
  <img
    alt="React TODO"
    className="sc-bZQynM jBlHCL"
    src="../../assets/logo.png"
  />
  <div
    className="sc-bdVaJa kTrVIl"
  >
    <input
      className="sc-bwzfXH fAHDEt"
      onChange={[Function]}
      placeholder="What needs to be done?"
      type="text"
      value=""
    />
    <button
      className="sc-htpNat kOKcXe"
      onClick={[Function]}
    >
      Create
    </button>
  </div>
  <table
    className="sc-ifAKCX jWxRiD"
  >
    <tbody />
  </table>
  <div
    className="sc-bxivhb IDrsk"
  >
    <ul>
      <li>
        "Each Todo list item has two state.
         
        <span
          style={
            Object {
              "color": "green",
            }
          }
        >
           Green
        </span>
        (Completed) and
        <span
          style={
            Object {
              "color": "red",
            }
          }
        >
           Red
        </span>
        (Incomplete)."
      </li>
      <li>
        Click on Todo List Description to toggle the state of todo
      </li>
    </ul>
  </div>
</div>
`;
